### PR TITLE
Cybersecurity Awareness Month

### PR DESCRIPTION
**Pull request by voltone**
* ~~Add landing page (not listed in the main index, but linked to directly in marketing communications)~~
* Add more OWASP references to the Web App Security Best Practices
* Remove "draft" from our main three documents

### CODE REVIEW

File: docs/landing_pages/owasp.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+title: EEF Security WG and OWASP
+layout: page
+---
+
+[![OWASP][owasp_logo]][owasp_website]
+
+The EEF Security Working Group and [OWASP][owasp_website] share a mission to
+encourage secure software development practices and to share practical,
+actionable knowledge with developers worldwide. While OWASP resources tend to
+be platform agnostic, the EEF Security WG focusses on the Erlang ecosystem,
```
Comment (maennchen) [inline docs/landing_pages/owasp.md]:
```suggestion
be platform agnostic, the EEF Security WG focuses on the Erlang ecosystem,
```

So far we're using american English in the guides i think.

File: docs/landing_pages/owasp.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+making the provided resources very much complementary.
```
Comment (maennchen) [inline docs/landing_pages/owasp.md]:
```suggestion
making the provided resources highly complementary.
```

**Review by maennchen — CHANGES_REQUESTED** [review]

File: docs/landing_pages/owasp.md (hunk 2 above)
Comment (voltone) [reply docs/landing_pages/owasp.md]: I think without very, in that case, no?

Comment (maennchen) [reply docs/landing_pages/owasp.md]: Yes. I was intending without very, but messed up the suggestion.

**Review by maennchen — APPROVED** [review]

**Review by maennchen — APPROVED** [review]

**Pull request merged**